kolibri/core/assets/src/views/core-base.vue

```diff
@@ -90,12 +101,24 @@
         type: Number,
         default: 0,
       },
+      // IMMERSIVE-SPECIFIC
+      immersivePage: {
+        type: Boolean,
+        required: false,
+      },
+      icon: {
+        type: String,
+        required: false,
+        default: 'close',
+      },
+      route: {
```

Comment:
Can we rename these props so it is clear they are specific to the immersive page action bar?

Reply:
You think that's necessary? 

I considered following a similar model to what you did in `snackbar`, but I thought the opaqueness of passing in an object wouldn't be worth the specificity. 

You're vying for something like `immersivePageRoute`?

Reply:
> You're vying for something like immersivePageRoute?

Yes. Because, the default app bar already has an icon (the hamburger). So we should make it clear that these props would only applicable when in immersivePage mode.